Sources/Common/Core/Math/index.js

```diff
@@ -1,6 +1,9 @@
 import seedrandom from 'seedrandom';
 import macro from 'vtk.js/Sources/macro';
 
+// eslint-disable-next-line import/no-cycle
```

Comment:
In general cycle are bad. Do you really need that cross dependency?

Reply:
Well..; I just wanted to simplified the code and use the Matrix builder. And the Matrix builder uses Math...
Do you have any idea on how I can make it better ?

Reply:
@jourdain sorry, I merged it before seeing your comment, we will fix it tomorrow